cmd/security_group_show.go

```diff
@@ -76,6 +85,31 @@ func (o *securityGroupShowOutput) toTable() {
 
 			return buf.String()
 		}
+
+		return "-"
+	}
+
+	formatInstances := func(instances []securityGroupInstanceOutput) string {
+		if len(instances) > 0 {
```

Comment:
Just a suggestion, you can flatten the code block if you first add:

```
if len(instances) == 0 {
    return "-"
}
...
```

IMO code is a bit easier to read that way.